el-tck/docs/userguide/src/main/jbake/content/using-examples.inc

```diff
@@ -22,9 +22,9 @@ sigtest-maven-plugin in the TCK runner as below.
 
 =======================================================================
 <plugin>
-    <groupId>org.netbeans.tools</groupId>
+    <groupId>jakarta.tck</groupId>
     <artifactId>sigtest-maven-plugin</artifactId>
-    <version>1.5</version>
+    <version>2.2</version>
```

Comment:
2.3 is probably a better release as it has better back compatibility.

Reply:
The issue with 2.3 is it breaks the `-IgnoreJDKClass` argument which _some_ signature test runners use. With that change you can only pass one `-IgnoreJDKClass` argument, which causes signature tests that pass it more than once to fail. In previous versions passing it for each type you want to ignore was the way to do it.

I think https://github.com/eclipse-ee4j/jakartaee-tck-tools/pull/40 somewhat undoes it, but IIRC it doesn't allow passing parameters to `-IgnoreJDKClass` which also breaks compatibility. 

It feels safer to stick with 2.2 for now.